src/router/index.js

```diff
@@ -137,7 +155,10 @@ router.beforeEach((to, from, next) => {
       'Chat',
       'CaseCreate',
       'Activity',
-      'CaseDetail'
+      'CaseDetail',
+      'CreateGroup',
```

Comment:
Remove `CreateGroup` from public route. You need to be authenticated to create one.

Reply:
It is done in groups after merging it will be solved !